server/start.go

```diff
@@ -56,6 +56,7 @@ const (
 	FlagIndexEvents       = "index-events"
 	FlagMinRetainBlocks   = "min-retain-blocks"
 	FlagIAVLCacheSize     = "iavl-cache-size"
+	FlagIAVLFastnode      = "iavl-fast-node"
```

Comment:
Does the flag name need to be the same as the config name?

Reply:
thanks, fixed